force_bdss/events/mco_events.py

```diff
@@ -47,7 +47,7 @@ class MCOFinishEvent(BaseDriverEvent, UIEventMixin):
     """ The MCO driver should emit this event when the evaluation ends."""
 
     def serialize(self):
-        raise NotImplementedError
```

Comment:
Could we put a docstring here: "We don't expect these events to carry any information that requires serialization"

Reply:
Can we do the coverage test locally before pushing the branch? Is there a python -m ci command?

Reply:
`python -m ci coverage`